app/src/main/java/Craigslist/clone/DBConnector.java

```diff
@@ -39,4 +44,118 @@ public int getNumListings() {
             throw new RuntimeException("Unable to get the number of listings.", e);
         }
     }
+
+    private Listing getListingFromResultSet(ResultSet resultSet){
```

Comment:
can this be a `static` method? As long as a method does not require any members (variables declared in the class), make them static.